intellij-plugin/src/main/kotlin/com/apollographql/ijplugin/inspection/Apollo4AvailableInspection.kt

```diff
@@ -36,11 +37,15 @@ class Apollo4AvailableInspection : LocalInspectionTool() {
   // XXX kts files are not highlighted in tests
   private val buildGradleFileName = if (isUnitTestMode()) "build.gradle.kt" else "build.gradle.kts"
 
+  // Do not warn until 4.0 is stable (but keep enabled always in unit tests)
+  private val isEnabled = isUnitTestMode() || !ApolloV3ToV4MigrationProcessor.apollo4LatestVersion.contains('-')
```

Comment:
This doesn't work for SNAPSHOTs like `4.0.1-SNAPSHOT` which is OK if you maintain `apollo4LatestVersion` manually. Alternative would be using something like https://github.com/swiftzer/semver to do proper semver but that's more involved. Your call.

Reply:
I'll keep as-is to very slightly (debatable!) err on Bill's razor's side

Reply:
Sounds good 👍 . Also, who's Bill? 👀

Reply:
Our friend good ol' Bill Occam!

Reply:
Aahh William! Or is he known under 2 names? Which would be quite ironic 🙃